packages/shared/src/hooks/useConditionalFeature.ts

```diff
@@ -21,32 +16,19 @@ export const useConditionalFeature = <T extends JSONValue>({
   feature: Feature<T>;
   shouldEvaluate: boolean;
 }): UseConditionalFeature<T> => {
-  const { user } = useContext(AuthContext);
   const { growthbook } = useGrowthBookContext();
   const { ready } = useFeaturesReadyContext();
-  const queryKey = generateQueryKey(
-    RequestKey.Feature,
-    user,
-    feature.id,
-    shouldEvaluate,
-  );
+  const isPending = !(shouldEvaluate && ready);
+  let value: WidenPrimitives<T> = feature.defaultValue as WidenPrimitives<T>;
```

Comment:
I think we should memoize this value as it can be an object sometimes.

Reply:
it will be stable since it is const in featureManagement